.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
+  merge_group:
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  prefetch:
+    runs-on: ubuntu-latest
```

Comment:
we do everything in the monorepo on debian bullseye (soon to be bookworm). sucks that there isn't widespread use of debian (from what I saw) in actions

Reply:
😔